### PR TITLE
Translate to Portuguese: examples/MovieFun REST

### PR DESCRIPTION
**Pull request by Daniel-Dos**
link to Jira:https://issues.apache.org/jira/browse/TOMEE-2828

### CODE REVIEW

File: examples/moviefun-rest/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+    @SuppressWarnings("unchecked")
+    public Set<Class<?>> getClasses() {
+        return new HashSet<Class<?>>(Arrays.asList(LoadRest.class, MoviesRest.class));
+    }
+}
+....
+
+=== LoadRest
+
+É um POJO que é mapeado para um URL raiz (`load`) com o anotação @Path e possui métodos Java para servir solicitações a este URL raiz e seus sub-URLs. Nesse caso, só tenha uma solicitação POST onde use um EJB (MoviesBean) injetado na sua classe para fazer um carregamento inicial de dados no banco de dados.
```
Comment (hbelmiro) [inline examples/moviefun-rest/README_pt.adoc:40]:
```suggestion
É um POJO que é mapeado para um URL raiz (`load`) com o anotação @Path e possui métodos Java para servir solicitações a este URL raiz e seus sub-URLs. Nesse caso, só tem uma solicitação POST onde usa um EJB (MoviesBean) injetado na sua classe para fazer um carregamento inicial de dados no banco de dados.
```

File: examples/moviefun-rest/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+
+    public void setRating(int rating) {
+        this.rating = rating;
+    }
+}
+....
+
+=== MoviesBean
+
+Este é o EJB de acordo com a anotação @Stateless. Ele usa a persistência da unidade "unidade de filme" para persistir filme de entidades.
```
Comment (hbelmiro) [inline examples/moviefun-rest/README_pt.adoc:239]:
```suggestion
Este é o EJB de acordo com a anotação @Stateless. Ele usa a persistência da unidade `movie-unit` para persistir entidades de filmes.
```

File: examples/moviefun-rest/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+        entityManager.createQuery("delete from Movie").executeUpdate();
+    }
+}
+....
+
+== Executando
+
+Running the example is fairly simple. In the ``moviefun-rest'' directory
+run:
+
```
Comment (hbelmiro) [inline examples/moviefun-rest/README_pt.adoc:322]: Remove those lines

File: examples/moviefun-rest/README_pt.adoc (hunk 1 above)
Comment (Daniel-Dos) [reply examples/moviefun-rest/README_pt.adoc]: done thanks

File: examples/moviefun-rest/README_pt.adoc (hunk 3 above)
Comment (Daniel-Dos) [reply examples/moviefun-rest/README_pt.adoc]: done thanks

File: examples/moviefun-rest/README_pt.adoc (hunk 2 above)
Comment (Daniel-Dos) [reply examples/moviefun-rest/README_pt.adoc]: done thanks

**Comment by Daniel-Dos** [comment]
thanks @hbelmiro  for review .

**Pull request merged**